### PR TITLE
Add responsibilities of GSoC participants

### PR DESCRIPTION
**Pull request by sims1253**

### CODE REVIEW

**Comment by jayvdb** [comment]
Please build in a very explicit period for completing the cEP , and even stipulate that doing coding during this period will be seen as very bad behaviour worthy of failing the student.

The more work done on the cEP, the more likely the project will succeed.

**Review by jayvdb — APPROVED** [review]

**Comment by jayvdb** [comment]
ack 2618bf9

**Comment by sims1253** [comment]
@rultor merge

File: community/gsoc_responsibilities.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+- Get comfortable with the community and work at coala.
+- Apply for a passport and visa if you want to meet coala at a conference and
+  need it.
+- Talk to your mentor if any problems arise.
+- Talk to an org admin if any problems with your mentor arise asap.
+
+
+#### Coding Phase
+
+- Follow your schedule. Communicate any problems with your mentor asap.
```
Comment (sils) [inline community/gsoc_responsibilities.md:55]: "Otherwise you'll be *FAILED*"

**Pull request merged**